### DISCUSSION TITLE
Java version detection from Minecraft version

### DISCUSSION THREAD

**Discussion by enorsu** [PrismLauncher/PrismLauncher#2649] [Ideas]
So, At the start you can set the path's to Java versions like 8, 17, 21
Then you don't have to manually set the Java EVERY SINGLE TIME, when creating an instance

**Comment by Trial97** [comment]
in 9.0 the java selection will be automatically done(based on minecraft version prism will use java 8,17 or 21) see: https://github.com/PrismLauncher/PrismLauncher/pull/2069
If you mean that you want to have different settings see this issue: https://github.com/PrismLauncher/PrismLauncher/issues/762

**Reply by enorsu** [reply]
when will it be released?

**Reply by Trial97** [reply]
The plan is to release the 9.0 in the first half of August(but it may be delayed if the milestone is not done)
